src/katello/agent/katelloplugin.py

```diff
@@ -162,7 +167,7 @@ def update_settings():
     # the client certs
     ca_candidates = [os.path.join(ca_cert_dir, 'katello-default-ca.pem'),
                      os.path.join(ca_cert_dir, 'candlepin-local.pem'),
-                     rhsm_conf['rhsm']['repo_ca_cert'] % rhsm_conf['rhsm']]
+                     rhsm_conf['rhsm']['repo_ca_cert'] % {'ca_cert_dir': ca_cert_dir}]
```

Comment:
Yes, @jlsherrill the python looks syntactically correct.
Side note: the removed line:
```
- rhsm_conf['rhsm']['repo_ca_cert'] % rhsm_conf['rhsm']]
```
should have raised an error if the value of `repo_ca_cert` is `%(ca_cert_dir)skatello-server-ca.pem` since the format does not match the substitution.